ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -6,6 +6,10 @@
 import glob
 import logging
 import shutil
+import sys
+if sys.platform == 'win32':
```

Comment:
Do you mind making this `if sys.platform.startswith('win'):`, this will be more future proof, especially if they change this string in future versions of platform. See [here](https://docs.python.org/2/library/sys.html#sys.platform).

Reply:
Fixed